packages/pharos/src/components/dropdown-menu/pharos-dropdown-menu.wc.stories.jsx

```diff
@@ -178,6 +178,36 @@ export const FullWidth = {
   },
 };
 
+export const OnBackground = {
+  render: () => {
+    const effect = () => {
+      useEffect(() => {
+        setTimeout(() => {
+          document.querySelector('#my-button-on-background').click();
+        }, 700);
+      });
+    };
+    effect();
```

Comment:
I think [Storybook might have nicer APIs](https://storybook.js.org/docs/react/essentials/interactions) for doing things like this now...something for later since I see you're borrowing an existing pattern, unless you're so inclined.